cypress/integration/v2/features/dynamic-content/current-post.spec.js

```diff
@@ -0,0 +1,326 @@
+		cy.setupWP()
+
+		const assertValue = value => selector().contains( containsRegExp( value ) ).should( 'exist' )
+
+		// Assert changing the Post Title
+		createNewPostWithCTA()
+		cy.typePostTitle( 'Dynamic Content Test' )
+		adjustField( 'Post Title' )
+		cy.getPostUrls().then( ( { editorUrl, previewUrl } ) => {
+			assertValue( 'Dynamic Content Test' )
+			cy.visit( previewUrl )
+			assertValue( 'Dynamic Content Test' )
+			cy.visit( editorUrl )
+			// Change the value of the Post Title
+			cy.typePostTitle( 'My Post' )
+			cy.visit( previewUrl )
+			// Assert the new value in the frontend.
+			assertValue( 'My Post' )
+		} )
+
+		// Assert changing the Post URL
+		createNewPostWithCTA()
+		adjustField( 'Post URL' )
+		cy.getPostUrls().then( ( { previewUrl } ) => {
+			cy.typePostTitle( 'My Post' )
+			cy.publish() // Publishing creates a new URL slug
+			cy.visit( previewUrl )
+			assertValue( `${ Cypress.config( 'baseUrl' ) }my-post/` )
+		} )
+
+		// Assert changing the Post Slug
+		createNewPostWithCTA()
+		cy.publish() // Publishing creates a post slug
+		adjustField( 'Post Slug' )
+		cy.getPostUrls().then( ( { previewUrl } ) => {
+			cy.addPostSlug( 'my-post-slug' )
+			cy.publish()
+			cy.visit( previewUrl )
+			assertValue( 'my-post-slug' )
+		} )
+
+		// Assert changing the Post Excerpt
+		createNewPostWithCTA()
+		cy.addPostExcerpt( 'Sample excerpt for this post.' )
+		adjustField( 'Post Excerpt' )
+		selector().contains( 'Sample excerpt for this post.' ).should( 'exist' )
+		cy.getPostUrls().then( ( { editorUrl, previewUrl } ) => {
+			cy.visit( previewUrl )
+			selector().contains( 'Sample excerpt for this post.' ).should( 'exist' )
+			cy.visit( editorUrl )
+			cy.addPostExcerpt( 'Lorem ipsum dolor sit amet.' )
+			cy.visit( previewUrl )
+			selector().contains( 'Lorem ipsum dolor sit amet.' ).should( 'exist' )
+		} )
+
+		// Assert changing the Post Status
+		createNewPostWithCTA()
+		adjustField( 'Post Status' )
+		cy.getPostUrls().then( ( { editorUrl, previewUrl } ) => {
+			cy.visit( previewUrl )
+			assertValue( 'draft' )
+			cy.visit( editorUrl )
+			cy.publish()
+			cy.visit( previewUrl )
+			assertValue( 'publish' )
+		} )
+
+		// Assert changing the Comment Status
+		createNewPostWithCTA()
+		adjustField( 'Comment Status' )
+		assertValue( 'open' )
+		cy.getPostUrls().then( ( { editorUrl, previewUrl } ) => {
+			cy.visit( previewUrl )
+			assertValue( 'open' )
+			cy.visit( editorUrl )
+			cy.editPostDiscussion( { 'Allow comments': false } )
+			cy.visit( previewUrl )
+			assertValue( 'closed' )
+		} )
+
+		// TODO: Assert changing Comment number, Featured image URL
+	} )
+}
+
+function assertEmptyValues() {
+	it( 'should assert empty values in the frontend', () => {
+		cy.setupWP()
+
+		const willAssertEmptyValue = () => {
+			cy.getPostUrls().then( ( { previewUrl } ) => {
+				cy.visit( previewUrl )
+				selector()
+					.should( $element => {
+						expect( $element.text().trim() ).equal( '' )
+					} )
+			} )
+		}
```

Comment:
do we need to still define the function?

Reply:
oh will remove this